domains/sajidportfolio.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Sajid-tech",
+        "email": "s.khan9430319425@gmail.com"
+    },
+    "record": {
+        "CNAME": "https://sajid-tech.github.io/sajidhussain.github.io"
```

Comment:
`CNAME` can't contain protocol.
`CNAME` can't contain `/`
(tip: rename your github website repo to `sajid-tech.github.io`)